src/custompresets.asm

```diff
@@ -0,0 +1,460 @@
+; --------------
+; Custom Presets
+; --------------
+
+print pc, " custom presets start"
+
+PresetSlot: ; 1AEh size slots
+    dw $0000, $01AE, $035C, $050A, $06B8, $0866, $0A14, $0BC2
+    dw $0D70, $0F1E, $10CC, $127A, $1428, $15D6, $1784, $1932
+    dw $1AE0, $1C8E, $1E3C, $1FEA, $2198, $2346, $24F4, $26A2
+    dw $2850, $29FE, $2BAC, $2D5A, $2F08, $30B6, $3264, $3412
+    dw $35C0, $376E, $391C, $3ACA, $3C78, $3E26, $3FD4, $4182
+    dw $4330, $44DE, $468C, $483A, $49E8, $4B96, $4D44
+
+custom_preset_save:
+{
+    LDA !sram_custom_preset_slot : ASL : TAX
+    LDA.l PresetSlot,X : TAX : LDY #$0000
+    LDA #$5AFE : STA $703000,X : INX #2 ; mark this slot as "SAFE" to load
```

Comment:
After this slot, the next slot would indicate how much space is actually used by the preset.  You had that at 1AEh but since we'd add two more bytes it would return 1B0h.  Theoretically we could use that information to selectively load only some of the information.

Right now we'd probably just ignore it with an INX #2 when loading a custom preset.